README.adoc

```diff
@@ -98,7 +98,7 @@ Use *spaces* for indentation. No hard tabs.
 Use 2 spaces to indent the bodies of
 forms that have body parameters.  This covers all `def` forms, special
 forms and macros that introduce local bindings (e.g. `loop`, `let`,
-`when-let`) and many macros like `when`, `cond`, `as->`, `cond->`, `case`,
+`when-let`) and many macros like `when`, `cond`, `as+++->+++`, `cond+++->+++`, `case`,
```

Comment:
Why the use of `+++`? Isn't it better to just surround the whole thing with `+` like `+as->+`? That looks much better to me.

Reply:
As you mentioned, it's more appropriate.
Sorry for my poor knowledge of AsciiDoc, I fixed it 42329a0ff0f0c3bf110393fc10d4db708ef3ebfd .